src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -249,6 +251,10 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
                 return throwError(() => error);
             }),
         );
+        // Start fetching, even if we don't subscribe to the result.
+        // This enables just calling this method to refresh the course, without subscribing to it:
+        this.loadCourseSubscription = observable.subscribe();
```

Comment:
If you unsubscribe in the `ngOnDestroy`, we should also check if `loadCourseSubscription` already contains a subscription from a previous refresh round, and unsubscribe from it. What do you think?

```suggestion
        this.loadCourseSubscription?.unsubscribe();
        this.loadCourseSubscription = observable.subscribe();
```